build.gradle.kts

```
@@ -3,10 +3,10 @@ description = "A Java client for KServe inference services."
 plugins {
     `java-library`
     id("net.researchgate.release") version "3.0.2"
```

Replace this with 
```suggestion
    id("com.bakdata.release") version "1.4.0"
```
and remove GitHubChangelogExtension config down below